lib/system/database/definitions/mysql.rb

```diff
@@ -87,14 +87,6 @@
     SQL
   end
 
-  trigger 'forums' do
```

Comment:
I am not sure how this works, but in case it creates triggers in the DB... should we remove the existing triggers via a migration?...

Reply:
Probably.. I don't know. I will try to make this PR only about UI and leave DB and backend stuff for later.